### ISSUE TITLE
Custom vtable dumper path

### ISSUE THREAD

**Issue by alexandernst** [lvc/abi-dumper#2]
Allow setting a custom path for the vtable dumper binary. `--vtable-dumper-path=` maybe?

**Comment by lvc** [comment]
Could you please describe why you need this option?

**Comment by alexandernst** [comment]
I'm really don't like installing (not even in `/opt`) tools for such a specific job. I rather keep them in my `~/git-projects` folder and use them from that path (which is not in my PATH).

**Comment by lvc** [comment]
Fixed in 0.99.10 release. Added option:

```
-vt-dumper PATH
    Path to the vtable-dumper executable if it is installed
    to non-default location (not in PATH).
```

Also the RAM memory usage has been greatly reduced when executing on large objects like Linux kernel (up to 80%).

Please update.
